airflow/www/static/js/graph.js

```diff
@@ -16,8 +16,8 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { generateTooltipDateTime, converAndFormatUTC, secondsToString } from './datetime-utils';
-import { escapeHtml } from './base';
+// global tiTooltip, taskTip
```

Comment:
Global as it's included in another file that is sourced.

If I used a normal import webpack would include it in two files.